### PR TITLE
Fix for the redirect object in SearchResponse

### PR DESCRIPTION
**Pull request by sfaiqh**

### CODE REVIEW

**Review by Haroenv — APPROVED** [review]
this seems correct, is it live?

File: packages/client-search/src/types/SearchResponse.ts [hunk 1]
```diff
@@ -224,6 +225,10 @@ export type SearchResponse<TObject = {}> = {
      * as a JSON object with one field per parameter.
      */
     params?: Record<string, any>;
+
+    redirect?: {
```
Comment (shortcuts) [inline packages/client-search/src/types/SearchResponse.ts:229]: if we have a link to the public doc, it would make sense to me to add it here

**Review by shortcuts — COMMENTED** [review]
same question as @Haroenv!

File: packages/client-search/src/types/SearchResponse.ts (hunk 1 above)
Comment (sfaiqh) [reply packages/client-search/src/types/SearchResponse.ts:229]: This parameter is not public and is not communicated to customers.

**Comment by sfaiqh** [comment]
> 



> this seems correct, is it live?

Yes

**Comment by sfaiqh** [comment]
@Haroenv anything else needed on this PR?

File: packages/client-search/src/types/SearchResponse.ts (hunk 1 above)
Comment (sfaiqh) [reply packages/client-search/src/types/SearchResponse.ts:232]: @shortcuts added a comment for internal use

**Review by shortcuts — APPROVED** [review]
looks good! :)

**Pull request merged**